src/gsi/gfs_stratosphere.f90

```diff
@@ -109,7 +109,8 @@ subroutine init_gfs_stratosphere
 
    use_gfs_stratosphere=.false.
    good_o3mr=.false.
-   nsig_max=120
+!   nsig_max=120
```

Comment:
just change this to 127, don't comment out 120.

Reply:
Changed to 127.